docs/installers.rst

```diff
@@ -379,10 +468,49 @@ Trying the installer locally
 ============================
 
 If needed (i.e. you didn't download the installer first from the website), add
-the ``runner`` and ``name`` directives. The value for ``runner`` must be the
-slug name for the runner. (E.g. winesteam for Steam Windows.)
-Save your script in a file and use the following command in a terminal:
-``lutris -i /path/to/file``
+the ``name``, ``game_slug``, ``slug`` and ``runner`` directives. The value for
+``runner`` must be the slug name for the runner. (E.g. winesteam for Steam Windows.)
+You can also add ``version``, ``description`` and ``notes`` to the installer file.
+Under ``script``, add ``files``, ``installer``, ``game`` and other installer
+directives. See below for an example.
+Save your script in a .yaml file and use the following command in a terminal:
+``lutris -i /path/to/file.yaml``
+
+Example Linux game:
```

Comment:
Not sure I want to advertise the API response format in the installer documentation.

Reply:
Why not? This is the only way to run installers locally.